outlook/src/taskpane/components/Helpdesk/TicketsSection/TicketsSection.tsx

```diff
@@ -55,8 +55,9 @@ class TicketsSection extends React.Component<TicketsSectionProps, TicketsSection
                 }
                 else
                 {
+                    const cids = this.context.getUserCompaniesString();
```

Comment:
Brackets before/after could be cleaned (updated code block, we may update it)

Reply:
Done.